Examples/HttpAnalyzer/HttpStatsCollector.h

```diff
@@ -151,7 +151,7 @@ class HttpStatsCollector
 
 		// verify packet is port 80
 		pcpp::TcpLayer* tcpLayer = httpPacket->getLayerOfType<pcpp::TcpLayer>();
-		if (!(tcpLayer->getTcpHeader()->portDst == htons(80) || tcpLayer->getTcpHeader()->portSrc == htons(80)))
+		if (!(tcpLayer->getTcpHeader()->portDst == htons(m_dstPort)))

```

Comment:
this change will cause HTTP response packets not to be counted. Please change is back to check the src port as well